app/schedulers/remove_unverified_users_cron_job.py

```diff
@@ -0,0 +1,23 @@
+from datetime import datetime, timedelta
+
+
+def complete_remove_unverified_users_job():
+    """
+    This function iterates of all the unverified users and
+    checks if email is not verified beyond a period of time.
+    if True then user is removed, if False does nothing
+    """
+    VERIFICATION_TIME = timedelta(days = 20)
+    from run import application
+    with application.app_context():
+        from app.database.models.user import UserModel
+        unverified_users = UserModel.query.filter_by(is_email_verified = False)
```

Comment:
Great idea! Querying by email verified before performing the loop here in python code 👏